di-ipv-cimit-stub/deploy/template.yaml

```diff
@@ -49,13 +49,48 @@ Resources:
         - SSMParameterReadPolicy:
             ParameterName: !Sub ${Environment}/stubs/*
 
+  # lambda to stub cimit postMitigations
+  PostMitigationsFunction:
+    Type: AWS::Serverless::Function
+    # checkov:skip=CKV_AWS_109: this requires a broad set of permissions
+    DependsOn:
+      - "PostMitigationsFunctionLogGroup"
+    Properties:
+      FunctionName: !Sub "postMitigations-${Environment}"
+      CodeUri: "../lambdas/post-mitigations"
+      Handler: uk.gov.di.ipv.core.postmitigations.PostMitigationsHandler::handleRequest
+      Runtime: java17
```

Comment:
could see java 11 in the build.gradle but here java 17.

Reply:
Java 17 Lambdas still kick out some warnings so safer to stick with Java 11 for now, methinks?

Reply:
I think, should be use to the same version java11 for Lambda

Reply:
Done.